src/vochain/vochain.proto

```diff
@@ -200,11 +216,61 @@ message AdminTx {
 }
 
 message RegisterKeyTx {
-        bytes nonce = 1;  // Unique number per vote attempt, so that replay attacks can't reuse this payload
-        bytes processId = 2;  // The process for which the vote is casted
-        Proof proof = 3;  // Franchise proof
-        bytes newKey = 4;   // New key to register
-        string weight = 5;   // Weight to delegate to newKey
+	bytes nonce = 1;  	  // Unique number per vote attempt, so that replay attacks can't reuse this payload
+	bytes processId = 2;  // The process for which the vote is casted
+	Proof proof = 3;  	  // Franchise proof
+	bytes newKey = 4;     // New key to register
+	string weight = 5;    // Weight to delegate to newKey
+}
+
+message MintTokensTx {
+	TxType txtype = 1;
+	uint32 nonce = 2;
+	bytes to = 3;
+	bytes value = 4;
+}
+
+message SendTokensTx {
+	TxType txtype = 1;
+	uint32 nonce = 2;
+	bytes from = 3;
+	bytes to = 4;
+	bytes value = 5;
+}
+
+message SetTransactionCostsTx {
+	TxType txtype = 1;
+	uint32 nonce = 2;
+	uint64 value = 3;
+}
+
+message SetAccountInfoTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	string infoURI = 3;
+}
+
+message SetAccountDelegateTx {
+	TxType txtype = 1;
+	bytes nonce = 2;
+	bytes delegate = 3;
+}
+
+message CollectFaucetTx {
+	TxType txType = 1;
+	bytes nonce = 2;
```

Comment:
I think we don't need a nonce here since the identifier of the FaucetPayload will avoid reusing the transaction.